feature-detects/css/calc.js

```diff
@@ -21,7 +21,7 @@ define(['Modernizr', 'createElement', 'prefixes'], function( Modernizr, createEl
   Modernizr.addTest('csscalc', function() {
     var prop = 'width:';
     var value = 'calc(10px);';
-    var el = createElement('div');
+    var el = createElement('a');
```

Comment:
Are you just changing div to be a because it's shorter? What about the fact that anchors are inline and divs are block this may affect some tests?